common/src/View/GamesPreferencePane.cpp

```diff
@@ -175,6 +175,9 @@ namespace TrenchBroom {
                 const std::string toolName = tool.name;
                 auto* edit = new QLineEdit();
                 edit->setText(IO::pathAsQString(gameFactory.compilationToolPath(m_gameName, toolName)));
+                if (tool.description) {
+                    edit->setToolTip(tool.description->c_str());
```

Comment:
Small nitpick, we normally use `QString::fromStdString(*tool.description)`. I think you're implicitly invoking this constructor https://doc.qt.io/qt-5/qstring.html#QString-7 - they both do the same thing, UTF-8 to QString, so it's just a consistency thing.